cc/defs.bzl

```diff
@@ -592,11 +592,15 @@ def swift_cc_test_library(**kwargs):
             be relative to the package this macro is called from.
     """
 
-    _ = kwargs.pop("nocopts", [])  # To handle API compatibility.
-
+    nocopts = kwargs.pop("nocopts", [])  # pop because nocopts is a deprecated cc* attr.
```

Comment:
Had a look at the rest of the file, and it seems `swift_c_test`  might also be missing the `standard` option? `swift_cc_test` uses it.